### DISCUSSION TITLE
Popover line-height

### DISCUSSION THREAD

**Discussion by johanneclc** [Orange-OpenSource/Orange-Boosted-Bootstrap#2403] [Q&A]
Hello, 

I have this button into my project which show popover and I would like to custom my line-height on popover

 ```html
<button
    type="button"
    class="btn btn-primary mx-3"
    [id]="'info' + i"
    data-bs-html="true"
    data-bs-placement="bottom"
    data-bs-title="Information"
    [attr.data-bs-content]="app['info' + lang]"
    (focus)="handlePopoverFocus('info' + i)"
    (blur)="handlePopoverBlur('info' + i)"
  >
   Title
  </button>
```

I try to add custom class with 

` data-bs-custom-class="custom-popover"`

with style : 
```css
.custom-popover {
  line-height: 0.2;
}
```

BUT with the  `data-bs-html="true"`, that's impossible. 

Thanks

**Comment by louismaximepiton** [comment]
Hey there,

If you're using Boosted, know that we don't recommand to change the default rendering we provide.
That said, it seems to [work well on our side without Angular](https://stackblitz.com/edit/ioahqx?file=index.html,index.js).
Could you provide any StackBlitz with the minimum code reproducing the error ?
